src/components/organisms/profile-tooltip/ProfileTooltip.css

```
@@ -0,0 +1,195 @@
+
+.tooltip-name {
+  margin-top: 2px;
+  font-size: 18px;
+  font-weight: 700;
+  max-width: 100%;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  transition: transform 0.2s ease;
+  padding: 0;
+  margin: 0;
```

Biome (2.1.2)

[error] Unexpected shorthand property margin after margin-top



(lint/suspicious/noShorthandPropertyOverrides)